brambling/views/utils.py

```diff
@@ -225,3 +228,77 @@ def get_context_data(self, **kwargs):
             'next_step': self.current_step.next_step if self.current_step else None,
         })
         return context
+
+
+class FinanceTable(object):
+
+    def __init__(self, event, transactions):
+        self.event = event
+        self.transactions = transactions
+
+    def headers(self):
+        return [
+            'Timestamp (%s)' % self.event.timezone,
+            'Created By',
+            'Method',
+            'Type',
+            'Order',
+            'Amount',
+            'Dancerfly Fee',
+            'Payment Fee',
+        ]
+
+    def header_row(self):
+        return [self.headers()]
+
+    def plaintext_rows(self):
```

Comment:
I think your instinct here was right. The explicit methods are probably an improvement over the magical iteration that happens in the ModelTables. We should probably port this over at some point. That being said, I wish there were a better naming convention than `plaintext_rows`... Maybe:

```python
def rows(self, include_headers=False):
    if (include_headers):
        # etc
```